blog/_posts/2021-03-15-scaling-inlets.md

```diff
@@ -0,0 +1,349 @@
+---
+layout: post
+title: Scaling inlets like a Pro with the help of Kubernetes
+description: Learn how to run tens or hundreds of secure inlets tunnels on a single Kubernetes cluster with the ability to scale for more.
+author: Johan Siebens
+tags: inlets-pro secure scaling tunnel kubernetes
+author_img: jsiebens
+image: /images/2021-03-scaling/background.jpg
+date: 2021-03-15
+---
+
+Learn how to run tens or hundreds of secure inlets tunnels on a single Kubernetes cluster with the ability to scale for more.
+
+## Introduction
+
+In a previous blog post, _[How to monitor multi-cloud Kubernetes with Prometheus and Grafana](https://inlets.dev/blog/2020/12/15/multi-cluster-monitoring.html)_, we demonstrated how we could use the inlets Pro Helm charts to bring multiple Prometheus instances into a single remote Kubernetes cluster, giving us a single plane of glass for monitoring all of them. The Helm charts proved to be very useful for setting up both the server part as the client part of an inlets tunnel. In the use case explained, we only tunnelled a couple of services, though. 
+
+So what if we want to bring a lot of services or applications into the cluster? Are the Helm charts available to assist is us? What do you need to know to scale to tens or hundreds of inlets tunnels?
```

Comment:
Give an example of why you may want to do this.

1) Service provider with an agent or managed service
2) A large organisation that has one cluster per tenant instead of a shared cluster